user/views.py

```diff
@@ -0,0 +1,78 @@
+import json
+from django.http        import JsonResponse
+from django.views       import View
+from .models            import Users
+from django.db.models   import Q
+
+class SignUp(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        
+        Users(
+            name        = data['name'],
+            email       = data['email'], 
+            password    = data['password'],
+        )
+        if '@' not in data['email'] or '.' not in data['email']:
+            return JsonResponse({'message':'EMAIL VALIDATION'},status =400)        
+        
+        if len(data['password']) < 8:
+            return JsonResponse({'message':'PASSWORD VALIDATION'},status=400)
+        
+        if Users.objects.filter(name = data['name']).exists() == True:
+            return JsonResponse({'message' : '이미 존재하는 아이디입니다.'},status=401)
+        else:
+            try:
+                Users.objects.create(name = data['name'], email = data['email'], password = data['password'])
+                return JsonResponse({'message': '회원가입완료'},status=200)
```

Comment:
```suggestion
                return JsonResponse({'message': '회원가입완료'},status=201)
```
정상적으로 데이터가 생성되었음을 알려주는 status code는 201 을 추천드립니다 👍